views/view-cloud.php

```diff
@@ -109,20 +105,12 @@ function do_personalities($personalities, $archetype, $dnsDomains, $sectionTitle
     }
 }
 
-
 //Actually Generating Page
 
 //First Infrastructure
-do_personalities($personalities, $archetype, Array('nubes.rl.ac.uk','stfc.ac.uk'), 'Infrastructure');
+do_personalities($personalities, $archetype, Array($CONFIG['NUBES']['URL_RL'], $CONFIG['STFC']['URL']), 'Infrastructure');

```

Comment:
This is ok for now, but it should actually a list of domains that are considered "part of the cloud".